setup.cfg

```diff
@@ -26,3 +26,7 @@ libnacl = asyncssh[libnacl]
 pkcs11 = asyncssh[python-pkcs11]
 pyopenssl = asyncssh[pyOpenSSL]
 pywin32 = asyncssh[pywin32]
+
+[options.entry_points]
+fsspec.specs =
+    ssh = sshfs.spec:SSHFileSystem
```

Comment:
This means that sshfs replaces sftpfs if installed, right?

Reply:
Yes, exactly.